main.go

```diff
@@ -25,7 +32,109 @@ var pongMessage = []byte(`{
 	]
 }`)
 
+type State struct {
+	db *sql.DB
+}
+
+var s *State
+
+func NewState(path string) (*State, error) {
+
+	// connection string
+	psqlconn := path
+
+	db, err := sql.Open("postgres", psqlconn)
+	if err != nil {
+		return nil, err
+	}
+
+	err = db.Ping()
+	if err != nil {
+		return nil, err
+	}
+
+	fmt.Println("Connected!")
+
+	s := &State{
+		db: db,
+	}
```

Comment:
See I comment I made on the last review about doing migrations. See also the linked repo I shared with you over Slack.

Reply:
I am not able to understand this comment. Can you please elaborate on this? Thank you :)

Reply:
Not sure I remember the context either here. I will resolve the convo.